modules/ROOT/pages/worker-monitoring.adoc

```diff
@@ -42,11 +42,17 @@ image::chmulesystemhealthmsg.png[CHMuleSystemHealthMsg]
 
 == How Monitoring Works
 
+*Runtime 3.x.y* 
+
 Every thirty seconds, CloudHub sends an HTTP get request to the ping flow within the application. If the application is running normally, the ping flow sends a response. If CloudHub receives no response from the ping three times in a row, the Logs print a warning and display a full thread dump.
 
 image::ch-monitoring-threaddump.png[CH-monitoring-threaddump]
 
-In addition, CloudHub triggers a notification to the console to alert you that a worker for this application stopped responding.
+*Runtime 4.x.y*
+
+Every thirty seconds, CloudHub checks the PID of the application. If CloudHub detects the application process is not running three times in a row, the Logs print a warning.
+
```

Comment:
I'm not sure what "any" application means.  This seems to be talking about a particular application (ie CloudHub checks the PID of "the" application). If we are talking about a particular app, then it should be as it was before..."this" application. If any means something else like a worker for all applications stopped responding, then let's use a different word.

Reply:
Can we say something like:
For Mule 3.x.x and 4.x.x apps, CloudHub triggers a notification...
?

Reply:
Or can we say: In both cases, CloudHub triggers a notification... ?

Reply:
@MichelleZMY That sounds good I think.